src/shared/components/workbasket-filters/workbasket-filters.component.html

```diff
@@ -0,0 +1,39 @@
+<h2 class="heading-h2" aria-label="Filters">Filters</h2>
```

Comment:
Should we not have a conditional logic to check the jursidcitions are loaded `isDataLoaded()` before the template can be displayed? This would be like we have on the other extracted components that pull data inside e.g. case-create.component.html or case-history.component.html to name a few.

Reply:
Done